test/modules/fetch/intercept/fetch.body.runtime.js

```diff
@@ -2,7 +2,7 @@ import { FetchInterceptor } from '@mswjs/interceptors/lib/interceptors/fetch'
 
 const interceptor = new FetchInterceptor()
 interceptor.on('request', (request) => {
-  window.requestBody = request.body
+  window.requestBody = request.text()
```

Comment:
We should await this, as `request.text()` returns a Promise.

Reply:
We already return promise for it in the test:

```
function getRequestBody(page: Page): Promise<string> {
  return page.evaluate(() => window.requestBody)
}
```

and documentation for `evaluate` says:

> If the function passed to the [page.evaluate(pageFunction[, arg])](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-browser/workbench/workbench.html) returns a [Promise], then [page.evaluate(pageFunction[, arg])](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-browser/workbench/workbench.html) would wait for the promise to resolve and return its value.